Tensile/Source/lib/source/ContractionProblem.cpp

```diff
@@ -913,9 +913,9 @@ namespace Tensile
 
 #ifdef TENSILE_USE_HALF
     template struct TypedContractionInputs<Half>;
+    template struct TypedContractionInputs<Half, Half, Half, Half, float, float>;
```

Comment:
Using this is going to fail.

HPA takes a half for alpha/beta for now.  See here:
https://github.com/ROCmSoftwarePlatform/Tensile/blob/653bda595bcfc043b59f97005a797a1bd638a748/Tensile/KernelWriterAssembly.py#L8041

Reply:
This could break hip-clang as well, given the mismatch with input arguments for ContractionProblem.cpp and ContractionProblem.hpp. I guess hip-clang uses an older Tensile commit for the time-being, but after branching, this may become an issue.

Reply:
The above change does not break anything. It simple adds another `ContractionInput`.

I'm asking that it be reinstated, since the tests pass.

Reply:
Regardless of kernels, we need structures to set up data structures with certain type combinations. This adds one which gets rid of a link error. It does not change the behavior of the library on other functions (other than the `call_once` changes, which are performance)

Reply:
The failures were in rocBLAS, not in Tensile